PSMDB-901 Release notes for 5.0.2-1 (#400)
percona/psmdb-docs: source/release_notes/5.0.2-1.rst

.. _PSMDB-5.0.2-1:

================================================================================

> *Percona Server for MongoDB* 5.0.2-1

Reviewer: Let's clearly mark it as "Release Candidate" here as well:
```suggestion
*Percona Server for MongoDB* 5.0.2-1 (Release Candidate)
```

Author: Added

================================================================================

:Date: August 16, 2021
:Installation: `Installing Percona Server for MongoDB <https://www.percona.com/doc/percona-server-for-mongodb/5.0/install/index.html>`_

We are pleased to announce the release candidate of Percona Server for MongoDB 5.0.2-1. 

> Percona Server for MongoDB 5.0.2-1 is an enhanced, open source, and highly-scalable database that is a

Reviewer: ```suggestion
Percona Server for MongoDB 5.0.2-1 is an enhanced, source-available, and highly-scalable database that is a
```

> fully-compatible, drop-in replacement for MongoDB 5.0.2 Community Edition. It  includes `all features of MongoDB 5.0.2 Community Edition <https://docs.mongodb.com/v5.0/release-notes/5.0/#5.0.2---aug-06--2021>`_ and provides :ref:`enterprise-level enhancements <compare>` for free.

Reviewer: Superfluous white space:
```suggestion
fully-compatible, drop-in replacement for MongoDB 5.0.2 Community Edition. It includes `all features of MongoDB 5.0.2 Community Edition <https://docs.mongodb.com/v5.0/release-notes/5.0/#5.0.2---aug-06--2021>`_ and provides :ref:`enterprise-level enhancements <compare>` for free.
```

Author: Fixed

> The following are most notable features introduced in MongoDB 5.0.2 Community Edition:

Reviewer: ```suggestion
Percona Server for MongoDB 5.0.2-1 includes all of the features of MongoDB 5.0 Community Edition. 

The most notable among these are:
```

Reviewer 2: But these are features in MongoDB CE not in Percona Server for MongoDB, current wording seems to be more accurate and it was previously used:
https://www.percona.com/doc/percona-server-for-mongodb/4.0/release_notes/4.0.4-1.html
https://www.percona.com/doc/percona-server-for-mongodb/4.2/release_notes/4.2.0-1.html

Reviewer: Well, by these being features of MongoDB CE, they are features of PSMDB as well. I've updated the proposed change, hopefully that makes it more clear.

- `Time Series Collections <https://docs.mongodb.com/v5.0/release-notes/5.0/#time-series-collections>`_ are aimed for storing sequences of measurements over a period of time. These collections improve query efficiency, allow data analysis in real time and optimize disk usage.
- `Window operators <https://docs.mongodb.com/v5.0/release-notes/5.0/#window-operators>`_ allow operations on a specified span of documents known as window. `$setWindowFields <https://docs.mongodb.com/v5.0/reference/operator/aggregation/setWindowFields/#mongodb-pipeline-pipe.-setWindowFields>`_ is a new pipeline stage to operate with these documents.
- `Auditing filters configuration at runtime <https://docs.mongodb.com/v5.0/release-notes/5.0/#auditing>`_
- `Versioned API <https://docs.mongodb.com/v5.0/reference/versioned-api/>`_ allows specifying which API version your application communicating with MongoDB runs against. Versioned API detaches the application's lifecycle from that of the database. As a result you modify the application only to introduce new features instead of having to maintain compatibility with the new version of MongoDB.

> MongoDB 5.0.2 Community Edition also adds `new aggregation operators <onfluence.percona.com>`_ such as ``$count``, ``$dateAdd``, ``$dateDiff``, ``$dateSubtract``, ``$sampleRate`` и ``$rand`` and introduces `mongosh <https://docs.mongodb.com/v5.0/release-notes/5.0/#new-mongodb-shell--mongosh>`_ as a new shell client.

Reviewer: The URL to "aggregation operators" seems to be incomplete?
Also, there's a stray `и` in the enumeration.

Author: Yes, you are right, sorry about that. Fixed

> MongoDB 5.0.2 Community Edition also adds `new aggregation operators <https://docs.mongodb.com/v5.0/release-notes/5.0/#new-aggregation-operators>`_ such as ``$count``, ``$dateAdd``, ``$dateDiff``, ``$dateSubtract``, ``$sampleRate`` and ``$rand``.

Reviewer: ```suggestion
Percona Server for MongoDB 5.0 also adds `new aggregation operators <https://docs.mongodb.com/v5.0/release-notes/5.0/#new-aggregation-operators>`_ such as ``$count``, ``$dateAdd``, ``$dateDiff``, ``$dateSubtract``, ``$sampleRate`` and ``$rand``.
```
